### ISSUE TITLE
Cyrillic

### ISSUE THREAD

**Issue by jemali-m** [weiweihuanghuang/Work-Sans#27]
Excellent print. Please add support for the Cyrillic alphabet. Thank you

**Comment by hpapazian** [comment]
Spongebob narrator voice: "Five... years... later..."

**Comment by sabihmir** [comment]
Any updates on this? Would love to have Mongolian Cyrillic as well 🙏

**Comment by Nikit** [comment]
Any updates?

**Comment by hpapazian** [comment]
Some designers only support Latin, while talking up a storm about "diversity"...

**Comment by weiweihuanghuang** [comment]
> Some designers only support Latin, while talking up a storm about "diversity"...

Some designers only comment in the github issues while talking up a storm about diversity. The project is open source. Why don’t you add  Cyrillic yourself Hrant 🙂

**Comment by hpapazian** [comment]
Some people have been supporting minority scripts since 1983 and get their students to do the same, and some people use open source as a cop-out and only reply to messages that concern their image. But we'd just be happy if somebody asking for non-Latin support wouldn't be ridiculed, and given the honest truth instead.

**Comment by Nikit** [comment]
Guys, this is very nice font, and adding Cyrillic will be very useful for a number of countries: 
https://en.wikipedia.org/wiki/Cyrillic_script
Thank you in advance!

p.s. This font is used in the template site on bootstrap 5: https://themewagon.github.io/drivin/

**Comment by weiweihuanghuang** [comment]
> Some people have been supporting minority scripts since 1983 and get their students to do the same, and some people use open source as a cop-out and only reply to messages that concern their image. But we'd just be happy if somebody asking for non-Latin support wouldn't be ridiculed, and given the honest truth instead.

That's great. I will point out that Hrant you are the one ridiculing and attacking me. I am asking you seriously why don't you add Cyrillic yourself – it is an open source project – why can't you give us the honest truth?

**Comment by hpapazian** [comment]
For one thing Cyrillic (being a €uropean Union requirement...) already has many good typefaces, so anybody who expects more from Culture than profitability would instead focus on under-served writing systems (see the talks by Craig Eliason and myself at TypeCon'23). But also this particular Latin typeface is frankly so generic that any suitable Cyrillic addition would also be quite forgettable.

The thing is you don't really have to make Cyrillic, Armenian or any other non-Latin fonts to earn the respect of people beyond the envious fawners, you just have to cut the hypocrisy that only serves the owners of the dominant culture. So next time somebody asks you to make a non-Latin, instead of ridiculing such a request please just be honest and reply "why would i care lol". That would be sad, but at least not worth the effort to confront. Because nothing erodes social justice like hypocrisy, and nothing promotes social justice in typography than supporting minority writing systems. The rest is all posturing that fronts personal fortune & glory, which needs to be confronted by anybody who cares.

**Comment by Nikit** [comment]
Hmm, soon this will off-topic, can you please talk about designer open-source issues somewhere?

**Comment by weiweihuanghuang** [comment]
Hrant you keep avoiding the question of why you don't make Cyrillic – is it becaus you don't care? 

I'll answer why I can't do it right now, I can't take 3–6 months off to work for free.

**Comment by weiweihuanghuang** [comment]
Hrant it seems very hypocritcal why you keep avoiding the question of why you don't make Work Sans Cyrillic – is it becaus you don't care? 

If Work Sans is so forgettable why are you in the comments attacking and ridiculing me with ad-hominem attacks to make Cyrillic for it. It's almost as if you don't care about actually supporting minority scripts and more about demonstrating a personal moral high ground.

I'll answer why I can't do it right now – I would love to extend Work Sans into as many scripts as possible. But I don't have the resources to take 3–6 months off to work for free on making Cyrillics alone. Extending it into other scripts to a high quality, if you aren't aware Hrant, would take even more time, possibly years.

**Comment by hpapazian** [comment]
As I said, it's because any time I can spare is better used elsewhere (like how I charge nothing for individuals seeking my feedback on their work). But the point is I would never ridicule somebody who asks for a non-Latin extension. Anyway don't worry about my own circumstances (a day job, four kids, etc.) and good/bad decisions, worry about transcending your own personal fortune & glory.

You can take three to six months over the nine years it's been. Pay your success forward by putting your diversity effort where your diversity mouth is. And in type, diversity is very poorly correlated with Latin type.

I care about confronting what distracts from true diversity. And I try to get others to care too. It's really not about you personally.

**Comment by weiweihuanghuang** [comment]
I understand now. Your own personal circumstances and time matters more than other people’s. And your contribution to diversity is attacking people because their time is less valuable than yours. Feel free to prove that it’s not true by making Work Sans Cyrillic, Greek, Armenian, Georgian, Arabic, Tamil, Telugu, Devanagari, etc over the next nine years :)

**Comment by hpapazian** [comment]
No, I am personally just as unimportant as you. It's what remains long after we die that counts, including what people we influence (like our students, or people who see our conference presentations) end up making. Culture.

How long do your Latin fonts take? A long time. If you don't have time to make a non-Latin now, when will you? When you retire? Is diversity something one supports only when idle? So please make any non-Latin font – and ideally not for a writing system that an economic union requires, but one that has few good fonts. Failing that, at least don't ridicule people who would like to see one made (which is how all this started) all the while pretending to care about diversity.

**Comment by alexburba** [comment]
I have created Cyrillic version for Regular and Bold styles

https://github.com/alexburba/Work-Sans-Cyr-Typeface
